plugin.video.tekthing/addon.xml

```diff
@@ -1,14 +1,16 @@
 ﻿<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <addon id="plugin.video.tekthing"
        name="Tekthing"
-       version="1.0.1"
+       version="1.0.2"
        provider-name="Skipmode A1">
   <requires>
-    <import addon="xbmc.python"                 version="2.1.0"/>
-    <import addon="xbmc.addon"                  version="12.0.0"/>
-	<import addon="script.module.beautifulsoup" version="3.0.8"/>
-	<import addon="script.module.requests"      version="2.4.3"/>
-    <import addon="plugin.video.youtube"        version="5.1.7"/>
+    <import addon="xbmc.python"                     version="2.1.0"/>
+    <import addon="xbmc.addon"                      version="12.0.0"/>
+	<import addon="script.module.beautifulsoup4"    version="4.5.3"/>
```

Comment:
@skipmodea1 ERROR: Version mismatch for addon script.module.beautifulsoup4. Required: 4.5.3, Available: 4.3.2